components/setting_picture.jsx

```diff
@@ -286,6 +261,55 @@ export default class SettingPicture extends Component {
             );
         }
 
+        let buttonRender;
+        if (this.props.onSubmit) {
+            buttonRender = (
+                <span>
+                    <input
+                        data-testid='uploadPicture'
+                        ref={this.selectInput}
+                        className='hidden'
+                        accept='.jpg,.png,.bmp'
+                        type='file'
+                        onChange={this.handleFileChange}
+                        disabled={this.props.loadingPicture}
+                        aria-hidden={true}
+                        tabIndex='-1'
+                    />
+                    <button
+                        data-testid='inputSettingPictureButton'
+                        className='btn btn-sm btn-primary btn-file sel-btn'
+                        disabled={this.props.loadingPicture}
+                        onClick={this.handleInputFile}
+                        aria-label={localizeMessage('setting_picture.select', 'Select')}
+                    >
+                        <FormattedMessage
+                            id='setting_picture.select'
+                            defaultMessage='Select'
+                        />
+                    </button>
+                    <button
+                        tabIndex={disableSaveButtonFocus ? '-1' : '0'}
+                        data-testid='saveSettingPicture'
+                        disabled={disableSaveButtonFocus}
+                        ref={this.confirmButton}
+                        className={confirmButtonClass}
+                        onClick={this.props.loadingPicture ? () => true : this.handleSave}
```

Comment:
The `() => true` function will create a new function every time causing unnecessary re-renders downstream vs creating a `this.returnTrue` method on the `SettingPicture` component.